pkg/piecestore/psserver/server.go

```diff
@@ -145,9 +145,9 @@ func (s *Server) Stop(ctx context.Context) error {
 	)
 }
 
-// Piece -- Send meta data about a stored by by Id
+// Piece -- Send meta data about a piece stored by Id
 func (s *Server) Piece(ctx context.Context, in *pb.PieceId) (*pb.PieceSummary, error) {
-	s.log.Debug("Getting Meta", zap.String("Piece ID", in.GetId()))
+	s.log.Info("Getting Meta", zap.String("Piece ID", in.GetId()))
```

Comment:
Try to keep one `Info` entry per call instead of multiple. So the entry-point could be `Debug` and when it exists with an error/successfully it would be `Info`.

Reply:
done